test/integration/vulnerability_scan_test.go

```diff
@@ -0,0 +1,159 @@
+// Copyright The Shipwright Contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package integration_test
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+
+	buildapi "github.com/shipwright-io/build/pkg/apis/build/v1beta1"
+	test "github.com/shipwright-io/build/test/v1beta1_samples"
+)
+
+var _ = Describe("Integration tests Build and BuildRuns", func() {
```

Comment:
```suggestion
var _ = Describe("Vulnerability scan tests", func() {
```

Reply:
Since it is in the `integration` package, I would say we do not need to repeat that in the name of the tests here.